platform/reworkd_platform/web/api/agent/views.py

```diff
@@ -12,6 +12,7 @@
     AgentSummarize,
     AgentChat,
     NewTasksResponse,
+    AgentResume,
```

Comment:
We currently do iterate based on Github actions, so you can set up Github actions for `mypy` and `pytest` although this is currently still an experimental feature.

I think this PR didn't work out too well since the retrieval step didn't do too well on this repo: will look more into this today.